test/io.jl

```diff
@@ -282,6 +282,25 @@ module TestIO
     @test df[:c5][2] == "true"
     @test df[:c5][3] == "true"
 
+    # Readtable column preselection
+    filename = "$data/typeinference/standardtypes.csv"
+    df = readtable(filename)
+    @test size(df) == (3,5)
+    df = readtable(filename, usecols=[1,3,5])
```

Comment:
A line break between each series of test wouldn't hurt. Also, you could mention `usecols` in the comment above. Finally, could you check for corner cases like providing duplicate column identifiers (should it work?), and invalid ones (use `@test_throws` to check that it fails)?